### DISCUSSION TITLE
Object properties should remain expanded

### DISCUSSION THREAD

**Discussion by jasonLaster** [RecordReplay/devtools#6990] [Ideas]
When you pause and expand an object, we should do our best to keep the object property expanded when you pause again and see the same object.

**Comment by jasonLaster** [comment]
Mark from glide helped me realize that it is okay to keep properties expanded by path, we don't need to do it on the object id

**Comment by bhackett1024** [comment]
Yeah, that's how I implemented this feature for the firefox devtools, I don't know where that logic ended up or if it's still usable though.

**Reply by jasonLaster** [reply]
Oh great - i bet it's still there. Will look in a bit.

Pretty sure @bvaughn will just replace it soon though. These old ObjectInspectors are gross

**Reply by markerikson** [reply]
Yeah, pretty sure BrianV's prototype has all the expanded paths working correctly.

**Reply by jasonLaster** [reply]
talking about objects in the scopes pane when you pause!
